src/test/java/org/cactoos/text/StrictTest.java

```diff
@@ -40,21 +38,20 @@
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
 public final class StrictTest {
 
-    /**
-     * A rule for handling an exception.
-     */
-    @Rule
-    public final ExpectedException exception = ExpectedException.none();
-
     /**
      * Ensures that Strict is failing on a negative predicate result.
      * @throws Exception If fails
      */
     @Test
-    public void failsIfPredicateIsNegative() throws Exception {
-        this.exception.expect(IllegalArgumentException.class);
-        this.exception.expectMessage(new StringContains("text"));
-        new Strict(s -> false, new TextOf("text")).asString();
+    public void failsIfPredicateIsNegative() {
+        MatcherAssert.assertThat(
```

Comment:
@atapin please use `Assertion` instead of the static methods from `MatcherAssert`